src/main/java/mate/jdbc/Main.java

```diff
@@ -1,7 +1,26 @@
 package mate.jdbc;
 
+import mate.jdbc.lib.Injector;
+import mate.jdbc.model.Car;
+import mate.jdbc.model.Driver;
+import mate.jdbc.model.Manufacturer;
+import mate.jdbc.service.CarService;
+
 public class Main {
+
+    private static final Injector injector = Injector.getInstance("mate.jdbc");
+
     public static void main(String[] args) {
-        // test your code here
+        Manufacturer manufacturer = new Manufacturer(2L, "Peugeot", "Hungary");
```

Comment:
don't pass id, id should be generated on saving. Generally, avoid hardcoded values as much as possible, instead of referencing it explicitly, call manufacturer.getId();